Cargo.toml

```diff
@@ -18,7 +18,7 @@ document-features = { version = "0.2", optional = true }
 encoding_rs = { version = "0.8", optional = true }
 serde = { version = "1.0.100", optional = true }
 tokio = { version = "1.0", optional = true, default-features = false, features = ["io-util"] }
-memchr = "2.0"
+memchr = "^2.1"
```

Comment:
The caret `^` isn't necessary, according to the reference it's equivalent to the default behavior.  Given this I would prefer it was kept consistent.

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements